src/core/cache.jl

```diff
@@ -18,7 +18,7 @@ function build_cache!(cache::TimeStatusChange, op_problem::OperationsProblem)
 end
 
 function build_cache!(cache::TimeStatusChange, psi_container::PSIContainer)
-    parameter = get_value(psi_container, cache.ref)
+    parameter = get_parameter_array(psi_container, cache.ref)
```

Comment:
This will assume that for any cache.ref the return type is a parameter array. This might not always be the case.

Reply:
What are the possible options?  I asked previously whether we needed to have a `get_value` function parameterized on UpdateRef.  The conclusion was `no`.  Also, in this case word `parameter` is used as a variable, so I assumed this was ok here.

Reply:
We can leave this as it is and put a note that parametrized get_value functions might be needed in the future. I don't have a specific case in mind yet but we would only be able to update Caches based on parameters and not variables or constraint duals.